monkey/common/events/stolen_credentials_event.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC
+from dataclasses import dataclass
+from ipaddress import IPv4Address
+from typing import FrozenSet, Iterable, Union
+from uuid import UUID
+
+from common.credentials import Credentials
+
+
+@dataclass(frozen=True)
+class StolenCredentialsEvent(ABC):
```

Comment:
```suggestion
class CredentialsStolenEvent(ABC):
```

I'm open to debate or other suggestions.

If I asked you "what was the event", or "what happened", you would say "Credentials were stolen." Hence `CredentialsStolenEvent` instead of `StolenCredentialsEvent`

Reply:
Just because of "Credentials were stolen" the name of the event that happen could be `StolenCredentialsEvent` maybe I am wrong with the grammar. What about `CredentialsTheftEvent` ?

Reply:
Naming things! It is just a sequence of Credentials. How about we just call it `CredentialsEvent`? Is there any reason we'd care whether or not they have been stolen?

Reply:
I think we care whether or not they were stolen. The stealing is the event. "Credentials" on their own don't constitute an event.

Reply:
I like `CredentialsStolenEvent`.
`StolenCredentialsEvent` makes it seem like stolen credentials did something which is being reported.

Reply:
Naming events in the past tense makes sense to me, because event objects are generated for events that already happened and because they happened. I like `CredentialsStolenEvent` better because it answers the question "what happened?" better. `Credentials[got]StolenEvent`.  `CredentialsTheftEvent` is better grammatically, but less natural since you need to convert an action to a noun. I'd say `NounVerbEvent` pattern is a bit better, but we could go for just `NounNounEvent`, the important part is to be consistent